source/ui/Components/Memory/MemoryComponent.qml

```diff
@@ -81,17 +84,44 @@ Item {
         }
     }
 
+
+    Component {
+        id: rowdelegate
+        // collapse row if cell is not needed
+        Rectangle{
+            height: (styleData.row % (number_bits / 8) == 0) ? 25 : 0
+        }
+    }
+
+    Component {
+        // disable row when cell is not needed
+        id: itemdelegate
+        Text {
```

Comment:
The QtQuickControls `Label` element might look better on certain systems (You just have to switch to `Label`, everything else should be the same).